PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## PR Description
```

Comment:
Can you rename to `.github/pull_request_template.md`? I don't want to add more config to the root if we can avoid it.

Reply:
ah thanks for catching that! did it!